lib/tasks/attempts.rake

```diff
@@ -0,0 +1,72 @@
+namespace :attempts do
+  AUTH_TOKEN = 'abc123'
+  PRIVATE_KEY_PATH = 'keys/attempts_api_private_key.key'
+
+  desc 'Retrieve events via the API'
+  task fetch_events: :environment do
+    conn = Faraday.new(url: 'http://localhost:3000')
+
+    resp = conn.post('/api/irs_attempts_api/security_events') do |req|
+      req.headers['Authorization'] = "Bearer #{AUTH_TOKEN}"
+    end.body
+
+    events = JSON.parse(resp)
+
+    if File.exist?(PRIVATE_KEY_PATH)
+      puts events['sets'].any? ? 'Decrypted events:' : 'No events returned.'
+
+      key = OpenSSL::PKey::RSA.new(File.read(PRIVATE_KEY_PATH))
+      events['sets'].each do |event|
+        begin
+          pp JSON.parse(JWE.decrypt(event[1], key))
```

Comment:
To me, arbitrary array index jumps out at me, where does `[1]` come from? Is this iterating over a (key, value) in a hash?

If so, we can do:
```suggestion
      events['sets'].each do |_jti, event|
        begin
          pp JSON.parse(JWE.decrypt(event, key))
```

Reply:
Fixed. Agreed, that was very ugly. This started out as a really rough local MVP sort of thing and it showed; thanks for helping to polish it up!